helper-script.py

```diff
@@ -0,0 +1,247 @@
+import argparse
+import logging
+import os
+import re
+import shutil
+import subprocess
+import typing
+
+import requests
+from github import Auth, Github
```

Comment:
Can we use our preferred style here, like `import github` and then `github.auth`, `github.Auth`?